src/microbe_stage/PlayerMicrobeInput.cs

```diff
@@ -11,6 +11,11 @@ public class PlayerMicrobeInput : Node
     /// </summary>
     private MicrobeStage stage;
 
+    /// <summary>
+    /// Whether or not the player is using auto-move
```

Comment:
I don't think this comment actually explains what this variable does. Also the xml element contents should be indented by 2 spaces.